x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/use_bulk_actions.ts

```diff
@@ -171,6 +176,49 @@ export const useBulkAddToCaseActions = ({
   ]);
 };
 
+export const useBulkUntrackActions = ({
+  setIsBulkActionsLoading,
+  refresh,
+  clearSelection,
+}: UseBulkUntrackActionsProps) => {
+  const onSuccess = useCallback(() => {
+    refresh();
+    clearSelection();
+  }, [clearSelection, refresh]);
+
+  const { application } = useKibana().services;
+  const { mutateAsync: untrackAlerts } = useBulkUntrackAlerts();
+  // Check if at least one Observability feature is enabled
+  if (!application?.capabilities) return [];
+  const hasApmPermission = application.capabilities.apm?.['alerting:show'];
```

Comment:
you need to add SLO and uptime. We should follow this list here `x-pack/plugins/observability/common/constants.ts`
```
export const observabilityAlertFeatureIds: ValidFeatureId[] = [
  AlertConsumers.APM,
  AlertConsumers.INFRASTRUCTURE,
  AlertConsumers.LOGS,
  AlertConsumers.UPTIME,
  AlertConsumers.SLO,
  AlertConsumers.OBSERVABILITY,
]; 
```

Reply:
Can't do a circular import of the observability plugin into triggersActionsUI unfortunately so we'll have to repeat it manually

Reply:
I am fine with that!